### PR TITLE
Add storage metadata

### LINKED ISSUE #24: Missing new metadata for storages
missing new metadata for storages from MaStR

### PR DESCRIPTION
**Pull request by Izzet91**
Closes #24 

Dear all,
please review this pull request. i added storage metadata json.

### CODE REVIEW

File: import-api/mastr_storage.json [hunk 1]
```diff
@@ -0,0 +1,162 @@
+{"name": "supply.bnetza_mastr",
+"title": "Bundesnetzagentur - Marktstammdatenregister - Stromerzeuger",
```
Comment (Ludee) [inline import-api/mastr_storage.json]: Stromspeicher

File: import-api/mastr_storage.json [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+            {"name": "timestamp_e", "description": "Timestamp (eeg)", "type": "timestamp", "unit": "none"} ],
+	 "primaryKey": "id"},
+    "dialect":
+        {"delimiter": ";",
+        "decimalSeparator": "."} } ],
+"review": {
+    "path": "https://github.com/OpenEnergyPlatform/data-preprocessing/issues/13",
+    "badge": "none"},
+"metaMetadata":
+    {"metadataVersion": "OEP-1.4",
```
Comment (Ludee) [inline import-api/mastr_storage.json:147]: It seems like the MaStR metadata is still with the "old" version 1.4. There was a final release as [OEP-1.4.0](https://github.com/OpenEnergyPlatform/examples/blob/master/metadata/oep_metadata_template.json#L80)
We will merge this PR and start a new branch to update all metadata to the newest version.

File: import-api/mastr_storage.json [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+    "schema": {
+        "fields": [
+            {"name": "EinheitMastrNummer", "description": "MaStR-Nummer der Einheit", "type": "string", "unit": "none"},
+            {"name": "id", "description": "ID der Einheit", "type": "int", "unit": "none"},
+            {"name": "lid", "description": "LID der Einheit", "type": "int", "unit": "none"},
+            {"name": "Name", "description": "Name der Einheit", "type": "string", "unit": "none"},
+            {"name": "Einheitart", "description": "Art der Einheit", "type": "string", "unit": "none"},
+            {"name": "Einheittyp", "description": "Typ der Einheit", "type": "string", "unit": "none"},
+            {"name": "Standort", "description": "Standort der Einheit", "type": "string", "unit": "none"},
+            {"name": "Bruttoleistung", "description": "Bruttoleistung der Einheiten", "type": "float", "unit": "none"},
```
Comment (Ludee) [inline import-api/mastr_storage.json]: unit is not none. It's "kW".

**Review by Ludee — CHANGES_REQUESTED** [review]
Looks good, some minor changes are needed

File: import-api/mastr_storage.json (hunk 1 above)
Comment (Bachibouzouk) [reply import-api/mastr_storage.json]: @Izzet91 did it in 8fd50a32a5e540c143e9de5258dd15051279de31

File: import-api/mastr_storage.json (hunk 3 above)
Comment (Bachibouzouk) [reply import-api/mastr_storage.json]: @Izzet91 did it in 8fd50a32a5e540c143e9de5258dd15051279de31

File: import-api/mastr_storage.json (hunk 2 above)
Comment (Bachibouzouk) [reply import-api/mastr_storage.json:147]: Created issue #43 for that :)

**Review by Bachibouzouk — APPROVED** [review]
You're good to go @Izzet91

**Pull request merged**